build/Linux+BSD/jessie-crosscompile-armhf.Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM debian:jessie
```

Comment:
Lets get it finished for Jessie and then you can adapt it for Wheezy if it's not too much effort.